xbmc/GUIInfoManager.cpp

```diff
@@ -1135,14 +1135,6 @@ const infomap weather[] =        {{ "isfetched",        WEATHER_IS_FETCHED },
 ///     @return **True** if Kodi is running on a linux/unix based computer.
 ///     <p>
 ///   }
-///   \table_row3{   <b>`System.Platform.Linux.RaspberryPi`</b>,
```

Comment:
Can you add this to removed infolabels for v19? Search for "@skinning_v18 **[Removed Infolabels]** The following infolabels have been removed:" in this file

Reply:
added, I hope I did that correctly!